### ISSUE TITLE
Angular Material 1.5.8 support, ngTouch warning message

### ISSUE THREAD

**Issue by angel1st** [corentin-gautier/angular-kaarousel#15]
Hi there,
I have two minor issues with this component and I would appreciate if you can provide some feedback on those:

1. Angular Material version support. Current version (2.0.5) supports Angular ~1.4.2, but the most recent Angular 1.x version is 1.5.8. It would be nice if it can be supported as well
1. `ngTouch` module - When started, Angular complain of `ngTouch` module with following warning: `You are using the ngTouch module. 
Angular Material already has mobile click, tap, and swipe support... 
ngTouch is not supported with Angular Material!`. Basically - you shouldn't need `ngTouch when using Angular Material.

Thanks,
Angel

**Comment by maximelafarie** [comment]
Yes but it's only an AngularJS warning message. It doesn't affect the way the component is working and it's useful for developers who wants to use this module without Angular Material. 😉
